File: pbwebrcewcl/src/app/appalti/commons/dto/get-procedure-aggiudicazione-request.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

/* tslint:disable */
import { FiltroProcedureAggiudicazione } from './filtro-procedure-aggiudicazione';
export interface GetProcedureAggiudicazioneRequest {
  filtro?: FiltroProcedureAggiudicazione;
  idProgetto?: number;
}

File: pbwebrce/pbwebrce-jar/src/main/java/it/csi/pbandi/pbwebrce/integration/request/GetProcedureAggiudicazioneRequest.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbwebrce.integration.request;

import it.csi.pbandi.pbwebrce.dto.FiltroProcedureAggiudicazione;

public class GetProcedureAggiudicazioneRequest {
	private FiltroProcedureAggiudicazione filtro;
	private Long idProgetto;
	public FiltroProcedureAggiudicazione getFiltro() {
		return filtro;
	}
	public void setFiltro(FiltroProcedureAggiudicazione filtro) {
		this.filtro = filtro;
	}
	public Long getIdProgetto() {
		return idProgetto;
	}
	public void setIdProgetto(Long idProgetto) {
		this.idProgetto = idProgetto;
	}
	
	
}
